kubernetes_tests/test_base.py

```diff
@@ -220,7 +220,7 @@ def ensure_dag_expected_state(self, host, logical_date, dag_id, expected_final_s
             print(f"Received: {result}")
             state = None
             for dag_run in result_json["dag_runs"]:
-                if dag_run["logical_date"] == logical_date:
+                if dag_run["run_after"] == logical_date:
```

Comment:
I don’t we should remove the `logical_date` key.

Reply:
Right. I missed it while changing it back. Thanks!